conf/ocp_version/ocp-4.5-ga-minus1-config.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Do these changes suppose to be part of this PR?

Reply:
No, fixed the rebase into squashing them